packages/create-atomic/src/plopfile.ts

```diff
@@ -1,6 +1,6 @@
 import {NodePlopAPI} from 'plop';
 import {spawn} from 'child_process';
-import {getPackageManager} from './utils';
+import {getPackageManager} from './utils.js';
```

Comment:
https://gist.github.com/sindresorhus/a39789f98801d908bbc7ff3ecc99d99c#how-can-i-make-my-typescript-project-output-esm
https://github.com/microsoft/TypeScript/issues/41887#issuecomment-741504718

Reply:
it is in TS, but we need to set it properly to the output with a .js in order for it to work with ES2020, like the links indicate. I'll try with nodenext, though